Pose2D position
string type
---
bool success
